.github/workflows/export-notion-blocks-and-commit.yml

```diff
@@ -15,15 +15,16 @@ jobs:
     timeout-minutes: 35
     steps:
       - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
+        with:
+          version: '7'
       - uses: actions/setup-node@v3
         with:
           node-version: '18'
-
-      - name: Install deps and build (with cache)
-        uses: bahmutov/npm-install@v1
+          cache: 'pnpm'
```

Comment:
cache from pnpm taken from example https://github.com/karlhorky/archive-webpage-browser-extension/blob/main/.github/workflows/lint-check-types-and-build.yml